exercises/binary-search/src/example/java/BinarySearch.java

```diff
@@ -0,0 +1,70 @@
+
+import java.util.List;
+
+public class BinarySearch<T extends Comparable<T>> {
+
+    public static String ARRAY_MUST_BE_SORTED = "Array should be sorted.";
```

Comment:
No problem! As far as I know, adding Javadocs to example implementations (when appropriate) should be just fine. Pinging @exercism/java for confirmation of that though.